benchmarks/src/jvmMain/kotlin/space/kscience/kmath/benchmarks/ExpressionsInterpretersBenchmark.kt

```diff
@@ -30,6 +31,19 @@ internal class ExpressionsInterpretersBenchmark {
     @Benchmark
     fun rawExpression(blackhole: Blackhole) = invokeAndSum(raw, blackhole)
 
+    @Benchmark
+    fun justCalculate(blackhole: Blackhole) {
```

Comment:
Add some kind of doc line to explain it is a **different** thing.

Reply:
OK